source/common/config/new_grpc_mux_impl.cc

```diff
@@ -10,17 +10,17 @@
 namespace Envoy {
 namespace Config {
 
-NewGrpcMuxImpl::NewGrpcMuxImpl(std::unique_ptr<SubscriptionStateFactory> subscription_state_factory,
```

Comment:
Can you rename the files also to get rid of new? (Here and in all the other files).

Reply:
+1